code/game/objects/items/devices/scanners/sequence_scanner.dm

```diff
@@ -20,6 +20,13 @@
 	var/list/buffer
 	var/ready = TRUE
 	var/cooldown = 200
+	var/list/genetic_makeup_buffer = list()
```

Comment:
Autodoc this var please
https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md